typescript/src/index.ts

```diff
@@ -14,6 +15,7 @@ module.exports = new Package('typescript', [require('../jsdoc')])
 // Register the services and file readers
 .factory(modules)
 .factory(exportSymbolsToDocsMap)
+.factory('tsHost', () => new Host())
```

Comment:
You can get away with a fat arrow function here, because it has no dependencies. But this would not work if you added a dependency as a parameter. Therefore I think it is safer to avoid using these when doing DI.